zerver/webhooks/groove/fixtures/ticket_assigned3.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Could you please give these fixtures names that are more descriptive instead of just numbering them? Just name them in terms of their differences.